tests/helpers/destroy-app.js

```diff
@@ -1,5 +1,6 @@
 import run from 'ember-runloop';
 
 export default function destroyApp(application) {
+    server.shutdown();
```

Comment:
Would you mind removing this line from the PR? I'd prefer if that change went in separately with some notes about why it's being made, especially as it's something that should be handled by `ember-cli-mirage`